kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/CommitterImpl.java

```diff
@@ -149,11 +149,10 @@ private void sendCommitResponse(UUID commitId, CommittableSupplier committableSu
               Event commitResponse =
                   new Event(
                       config.controlGroupId(),
-                      EventType.COMMIT_RESPONSE,
-                      new CommitResponsePayload(
+                      new DataWritten(
                           writerResult.partitionStruct(),
                           commitId,
-                          TableName.of(writerResult.tableIdentifier()),
+                          TableReference.of(config.catalogName(), writerResult.tableIdentifier()),
```

Comment:
@bryanck double checking: this should be `config.catalogName()` or should I be getting the actual catalog object and doing `catalog.name()` ?

Reply:
I felt the config value was safer, in case the catalog implementation returns a different name. That being said, we don't use the value currently.